python_client/tests/test.py

```diff
@@ -0,0 +1,66 @@
+import metadata
+import unittest
+
+class TestMetedata(unittest.TestCase):
+
+  def test_log_metadata(self):
+    ws1 = metadata.Workspace(backend_url_prefix="127.0.0.1:8080",
+                            name="ws_1",
+                            description="a workspace for testing",
+                            labels={"n1": "v1"})
+    r = metadata.Run(
+        workspace=ws1,
+        name="first run",
+        description="first run in ws_1",
+    )
+
+    r.log_data_set(metadata.DataSet(
```

Comment:
log(metadata.DataSet ? feels less verbose this way.

Reply:
Yes, log() is less verbose. But there is no function overloading in Python and I don't want to have type switch in log() now.

I plan later to abstract an common interface for these artifacts objects, like `is_run_input()`, `serialize_to_json()`. so that we can have log() works on various types.